### ISSUE TITLE
[JSON] StorageManagement returns tuples in getCacheReplicas and getStagedReplicas

### ISSUE THREAD

**Issue by chaen** [DIRACGrid/DIRAC#4322]
```
============================================= Decoding tuples =============================================
  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/scripts/dirac-agent.py", line 45, in <module>
    agentReactor.go()

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Base/AgentReactor.py", line 146, in go
    timeToNext = self.__scheduler.executeNextTask()

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/ThreadScheduler.py", line 134, in executeNextTask
    self.__executeTask( taskId )

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/ThreadScheduler.py", line 187, in __executeTask
    task[ 'func' ]( *task[ 'args' ] )

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Base/AgentModule.py", line 365, in am_go
    cycleResult = self.__executeModuleCycle()

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Base/AgentModule.py", line 430, in __executeModuleCycle
    result = self.am_secureCall(executors[0][0], executors[0][1])

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Base/AgentModule.py", line 327, in am_secureCall
    result = functor(*args)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/StorageManagementSystem/Agent/RequestPreparationAgent.py", line 33, in execute
    res = self.__getNewReplicas()

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/StorageManagementSystem/Agent/RequestPreparationAgent.py", line 135, in __getNewReplicas
    res = self.stagerClient.getCacheReplicas( {'Status':'New'} )

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Base/Client.py", line 132, in func
    return self.executeRPC(*args, **kwargs)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Base/Client.py", line 83, in executeRPC
    return getattr(rpcClient, toExecute)(*parms)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/RPCClient.py", line 43, in __call__
    return self.__doRPCFunc( self.__remoteFuncName, args )

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/RPCClient.py", line 90, in __doRPC
    return self.__innerRPCClient.executeRPC( sFunctionName, args )

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/private/InnerRPCClient.py", line 69, in executeRPC
    receivedData = transport.receiveData()

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/private/Transports/BaseTransport.py", line 253, in receiveData
    data = DEncode.decode(data)[0]

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/DEncode.py", line 405, in decode
    return g_dDecodeFunctions[data[0]](data, 0)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/DEncode.py", line 378, in decodeDict
    oD[k], i = g_dDecodeFunctions[data[i]](data, i)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/DEncode.py", line 340, in decodeTuple
    printDebugCallstack('Decoding tuples')

Calling frame: ('/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/private/Transports/BaseTransport.py', 253)
With arguments  {'data': 'ds2:OKb1s14:ParameterNamesls9:ReplicaIDs4:Types6:Statuss2:SEs3:LFNs3:PFNs4:Sizes12:FileChecksums4:GUIDs10:SubmitTimes10:LastUpdates6:Reasons5:Linkses7:Recordstes5:Valuedee'}
====================================================================================================
```

```
============================================= Decoding tuples =============================================
  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/scripts/dirac-agent.py", line 45, in <module>
    agentReactor.go()

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Base/AgentReactor.py", line 146, in go
    timeToNext = self.__scheduler.executeNextTask()

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/ThreadScheduler.py", line 134, in executeNextTask
    self.__executeTask( taskId )

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/ThreadScheduler.py", line 187, in __executeTask
    task[ 'func' ]( *task[ 'args' ] )

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Base/AgentModule.py", line 365, in am_go
    cycleResult = self.__executeModuleCycle()

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Base/AgentModule.py", line 430, in __executeModuleCycle
    result = self.am_secureCall(executors[0][0], executors[0][1])

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Base/AgentModule.py", line 327, in am_secureCall
    result = functor(*args)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/StorageManagementSystem/Agent/StageRequestAgent.py", line 36, in execute
    return self.submitStageRequests()

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/StorageManagementSystem/Agent/StageRequestAgent.py", line 69, in submitStageRequests
    res = self._getMissingReplicas()

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/StorageManagementSystem/Agent/StageRequestAgent.py", line 116, in _getMissingReplicas
    res = self.__getStagedReplicas()

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/StorageManagementSystem/Agent/StageRequestAgent.py", line 292, in __getStagedReplicas
    res = self.stagerClient.getStagedReplicas()

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Base/Client.py", line 136, in func
    return self.executeRPC(**kwargs)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Base/Client.py", line 83, in executeRPC
    return getattr(rpcClient, toExecute)(*parms)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/RPCClient.py", line 43, in __call__
    return self.__doRPCFunc( self.__remoteFuncName, args )

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/RPCClient.py", line 90, in __doRPC
    return self.__innerRPCClient.executeRPC( sFunctionName, args )

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/private/InnerRPCClient.py", line 69, in executeRPC
    receivedData = transport.receiveData()

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/private/Transports/BaseTransport.py", line 253, in receiveData
    data = DEncode.decode(data)[0]

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/DEncode.py", line 405, in decode
    return g_dDecodeFunctions[data[0]](data, 0)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/DEncode.py", line 378, in decodeDict
    oD[k], i = g_dDecodeFunctions[data[i]](data, i)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/DEncode.py", line 340, in decodeTuple

Calling frame: ('/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/private/Transports/BaseTransport.py', 253)
With arguments  {'data': 'ds2:OKb1s14:ParameterNamesls9:ReplicaIDs4:Types6:Statuss2:SEs3:LFNs3:PFNs4:Sizes12:FileChecksums4:GUIDs10:SubmitTimes10:LastUpdates6:Reasons5:Linkses7:Recordstes5:Valuedee'}
====================================================================================================

```

**Comment by chaen** [comment]
In https://github.com/DIRACGrid/DIRAC/pull/4366

**Comment by chrisburr** [comment]
I think this can be closed now #4410 has been merged?
